docs/pages/enroll-resources/enroll-resources.mdx

```diff
@@ -1,32 +1,265 @@
+<EnrollmentMethods>
+<Method
+title="Applications"
+icon={applicationSvg}
+href="./application-access/"
+tagLists={
+  [
+    {
+      tags: 
+        [
+          {
+            name: "Cloud consoles",
+            href: "./application-access/cloud-apis/",
+            icon: "cloud3",
+          },
+          {
+            name: "Web apps",
+            href: "./application-access/guides/connecting-apps/",
+            icon: "appWindow",
+          },
+          {
+            name: "TCP apps",
+            href: "./application-access/guides/tcp/",
+            icon: "tcp",
+          },
+          {
+            name: "more",
+            href: "./application-access/guides/",
+            arrow: true,
+          }
+        ] 
+    },
+  ]
+}
+>
+Secure access to web applications, TCP apps, cloud provider consoles and CLIs, and more.
+</Method>
+<Method
+title="Databases"
+icon={databaseSvg}
+href="./database-access/"
+tagLists={
+  [
+    {
+      tags: 
+        [
+          {
+            name: "AWS",
+            href: "./database-access/enroll-aws-databases/",
+            icon: "aws",
+          },
+          {
+            name: "Azure",
+            href: "./database-access/enroll-azure-databases/",
+            icon: "azure",
+          },
+         {
+            name: "Google Cloud",
+            href: "./database-access/enroll-google-cloud-databases/",
+            icon: "googleCloud",
+          },
+          {
+            name: "more",
+            href: "./database-access/",
+            arrow: true,
+          }
+        ] 
+    },
+  ]
+}
+>
+Secure, auditable access to cloud- and self-hosted databases as well as DBaaS.
+</Method>
+<Method
+title="Kubernetes Clusters"
+icon={kubernetesClustersSvg}
+href="./kubernetes-access/"
+tagLists={
+  [
+    {
+      title: "Kubernetes Clusters Guides",
+      tags: 
+        [
+          {
+            name: "Enroll Kubernetes clusters",
+            href: "./kubernetes-access/getting-started/",
+          },
+          {
+            name: "Access controls",
+            href: "./kubernetes-access/controls/",
+          },
+        ] 
+    },
+  ]
+}
+>
+Protect Kubernetes clusters with SSO integration, RBAC, and session recordings.
+</Method>
+<Method
+title="Windows Desktop"
+icon={windowsDesktopsSvg}
+href="./desktop-access/"
+tagLists={
+  [
+    {
+      title: "Windows Desktops Access Guides",
+      tags: 
+        [
+          {
+            name: "Local Windows users",
+            href: "./desktop-access/getting-started/",
+          },
+          {
+            name: "Active Directory",
+            href: "./desktop-access/active-directory/",
+          },
+          {
+            name: "Directory sharing",
+            href: "./desktop-access/directory-sharing/",
+          },
+        ] 
+    },
+  ]
+}
+>
+Passwordless access to Windows Desktops and servers for domain users and local users.
+</Method>
+<Method
+title="Linux Servers"
```

Comment:
I would consider calling this SSH Servers instead.
```suggestion
title="SSH Servers"
```

Here's a couple reasons why:
- Our server access integrations can run on non-Linux OSes like macOS and even some other Unix-like systems.
- We'll support remote desktop access on Linux next year, and while that is "Linux Servers" those docs won't fall under this section since it's all about SSH.